### ISSUE TITLE
Renamed packages installed by Yarn are not supported

### ISSUE THREAD

**Issue by maxcnunes** [snyk/snyk#330]
- `node -v`: v8.9.4
- `npm -v`: 6.4.1
- `snyk -v`: 1.122.0
- Command run: `./node_modules/.bin/snyk wizard`

I have a project that used to use bower and it was migrated to use yarn instead, here is the package.json

```json
{
  "name": "yarn-bower-snyk",
  "version": "1.0.1",
  "main": "server/app.js",
  "dependencies": {
    "@bower_components/angular": "angular/bower-angular#~1.4.0",
    "@bower_components/angular-animate": "angular/bower-angular-animate#~1.4.0",
    "@google/maps": "^0.3.0",
    "async": "^2.1.4",
    "snyk": "^1.122.0"
  },
  "devDependencies": {
    "grunt": "~0.4.5",
    "grunt-angular-templates": "^0.5.4",
    "grunt-babel": "^6.0.0"
  },
  "scripts": {
    "start": "node server",
    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'client\bower_components', 'junction') } catch (e) { }\""
  },
  "private": true
}
```

When I try to use snyk commands in this project like `./node_modules/.bin/snyk wizard` it fails:

```
Analyzing npm dependencies for /Users/maxnunes/Development/yarn-bower-snyk
/Users/maxnunes/Development/yarn-bower-snyk/node_modules/angular/package.json is not a node project
```

Looks like for dependencies with scope prefix like this `@bower_components/angular` snyk is removing the scope name prefix `@bower_components` and is looking for the dependency directly in node_modules `node_modules/angular` but the right path is `node_modules/@bower_components/angular`.

---

If applicable, please append the `--debug` flag on your command and include the output here **ensuring to remove any sensitive/personal details or tokens.

[log-debug.txt](https://github.com/snyk/snyk/files/2746837/log-debug.txt)

**Comment by lili2311** [comment]
Hi @maxcnunes ty for raising this, will look at scheduling this fix in shortly and let you know once ready!

**Comment by kyegupov** [comment]
@maxcnunes your `package.json` file doesn't seem right. This:

`"@bower_components/angular": "angular/bower-angular#~1.4.0",`

is not a valid syntax, there's no tag "~1.4.0" in that GitHub repo. 

It could be `angular/bower-angular#semver:~1.4.0`, but there's still no matching version of bower-angular, as far as I can see.

**Comment by kyegupov** [comment]
@maxcnunes also note that you are using backslash in `client\bower_components` which renders the path incorrect.

**Comment by kyegupov** [comment]
So, the problem is actually github repo imported via Yarn using an alias.
Minimum repro:

package.json:
```{
  "name": "dummy",
  "version": "0.0.1",
  "dependencies": {
    "shmoosync": "caolan/async"
  }
}


$ yarn install

$ snyk wizard

.../dummy/node_modules/async/package.json is not a node project
```

**Comment by kyegupov** [comment]
Should be fixed in Snyk CLI 1.143
